<launch>

  <arg name="marker_size" default="5.6" />
  <arg name="max_new_marker_error" default="0.08" />
  <arg name="max_track_error" default="0.2" />
  <arg name="cam_image_topic" default="/cameras/head_camera/image" />
  <arg name="cam_info_topic" default="/cameras/head_camera/camera_info_std" />
  <arg name="output_frame" default="/head_camera" />
  <arg name="mname" default="head_camera" />
  <arg name="marker_margin" default="2" />
  <arg name="max_freq" default="20" />
  <arg name="marker_resolution" default="5" />


  <arg name="resolution" default = "--resolution 1280x800" />
  <arg name="open" default = "--open head_camera" />

  <node name="head_cam" pkg = "baxter_tools" type="camera_control.py" args="$(arg open) $(arg resolution)" />

  <node name="ar_track_alvar0" pkg="ar_track_alvar" type="individualMarkersNoKinect" respawn="false" output="screen" args="$(arg marker_size) $(arg max_new_marker_error) $(arg max_track_error) $(arg cam_image_topic) $(arg cam_info_topic) $(arg output_frame) $(arg max_freq) $(arg marker_resolution) $(arg marker_margin)" />



</launch>